master/internal/user/service.go

```diff
@@ -119,6 +153,11 @@ func (s *Service) postLogout(c echo.Context) (interface{}, error) {
 }
 
 func (s *Service) postLogin(c echo.Context) (interface{}, error) {
+	if s.extConfig.JwtKey != "" {
```

Comment:
need a similar error to the this one and the one in grpc for patch user password below, i think.